File: 1-prime-numbers/solution.c
#include <stdio.h>

int main(void) {
	// your code goes here
	int number = 1000;
  int primenNumber = 0;
  if (number > 0) {
    primenNumber = 1;
  }

  for (int i = (number-1); i > 1; i--) {
    if(number % i == 0){
      primenNumber = 0;
      break;
    }
  }

  if (primenNumber == 1) {
    printf("%s\n", "Number is prime");
  } else {
    printf("%s\n","number is not prime");
  }


	return 0;
}

File: 1-prime-numbers/solution.js
// number to check
const number = 10000000000000000000;
let primenNumber = null;
// initial state of number, given number must be greater than zero
if (number > 0) {
  primenNumber = true;
}

for (var i = (number-1); i > 1; i--) {
  if(number % i === 0){
    console.log('entered loop');
    primenNumber = false;
    break;
  }
}


if (primenNumber === true) {
  console.log('Number is prime');
} else {
  console.log(`number is not prime`);
}
